meta-cli/src/deploy/actors/task/command/typescript.rs

```diff
@@ -17,12 +17,17 @@ pub(super) async fn get_raw_command(path: impl AsRef<Path>) -> Result<Command> {
         TsRuntime::Deno => {
             log::debug!("loading typegraph using deno");
             let mut command = Command::new("deno");
+
+            command.arg("run");
+            if let Ok(import_map) = std::env::var("MCLI_DENO_IMPORT_MAP") {
```

Comment:
**thought**: instead of a specialized param like this, wouldn't it be better to have a `MCLI_LOADER_TS`? If they want to specify an import map, they can set it to `deno --import-map=... --other-flag=...`.

Reply:
yeah that's a good idea. A question though, wouldn't that look like `MCLI_LOADER_TS="deno run -A run"` for example? Which sort of implies that we need to document somewhere it is the command prefix so the filename comes after.. But I guess we can introduce templates with predefined placeholders such as `{filename}` or `{cwd}`.